src/include/sof/lib/clk.h

```diff
@@ -41,12 +42,15 @@ struct clock_info {
 	uint32_t notification_mask;
 	spinlock_t lock;
 	int (*set_freq)(int clock, int freq_idx);
+	void (*low_power_mode)(int clock, bool enable);
```

Comment:
Can you comment what this means and how its different from setting a lower frequency.

Reply:
updated